pkg/motor/create_account.go

```diff
@@ -226,6 +234,15 @@ func createVault(mtr *motorNodeImpl, request mt.CreateAccountWithKeysRequest) {
 	// update DID Document
 	mtr.DIDDocument.AddService(vaultService)
 
+	if err != nil {
```

Comment:
`err` isn't being reassigned between now and the previous check, so this will always be false.